web/packages/teleport/src/components/AuthnDialog/AuthnDialog.tsx

```diff
@@ -41,13 +41,12 @@ export default function AuthnDialog({
           </Danger>
         )}
         <Text textAlign="center">
-          Re-authentication is required. Follow the prompts given by your
-          browser to complete authentication.
+          Re-enter your multi-factor authentication in the browser to continue.
         </Text>
       </DialogContent>
       <DialogFooter textAlign="center">
-        <ButtonPrimary onClick={onContinue} mr={3} width="130px">
-          {errorText ? 'Retry' : 'Verify'}
+        <ButtonPrimary onClick={onContinue} setRef={e => e && e.focus()} mr={3} width="130px">
```

Comment:
`autoFocus` should be used:
```suggestion 
        <ButtonPrimary onClick={onContinue} autoFocus mr={3} width="130px">
```

Reply:
Nice, thanks!